app/helpers/supplier_helper.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+module SupplierHelper
+  def supplier_link(supplier)
+    supplier.instance_of?(LeadProvider) ? nil : admin_delivery_partner_path(supplier)
```

Comment:
This will change when we have something to show for lead providers

Reply:
instance_of reads a bit like java. Can we use is_a?

Reply:
I think this is missing a unit test as well.